ovnnbimp.go

```diff
@@ -133,6 +133,22 @@ func (odbi *ovnDBImp) getRowUUIDContainsUUID(table, field, uuid string) (string,
 	return "", ErrorNotFound
 }
 
+func (odbi *ovnDBImp) getRowsMatchingUUID(table, field, uuid string) ([]string, error) {
+	odbi.cachemutex.Lock()
+	defer odbi.cachemutex.Unlock()
+	var uuids []string
+	for id, drows := range odbi.cache[table] {
+		v := fmt.Sprintf("%s", drows.Fields[field])
```

Comment:
why can not add this condition in delete transaction directly?

Reply:
for simplicity. Easier for review and understanding the logic